inspirehep/modules/authors/views/holdingpen.py

```diff
@@ -314,16 +314,23 @@ def newreview():
     objectid = request.values.get('objectid', 0, type=int)
     if not objectid:
         abort(400)
-    workflow_metadata.extra_data['is-update'] = True
+
     workflow_metadata = WorkflowUIRecord.get_record(objectid)['metadata']
 
+    # Initialise extra_data
```

Comment:
No need, I can rebase on top of this (the smaller the pr the better). If you were getting it empty, maybe we should do the extra check and initialize only if not there :/, something like:
```
workflow_metadata['extra_data'] = workflow_metadata.get('extra_data', {})
```